staging/src/k8s.io/client-go/tools/clientcmd/client_config.go

```diff
@@ -53,7 +53,7 @@ var (
 // getDefaultServer returns a default setting for DefaultClientConfig
 // DEPRECATED
 func getDefaultServer() string {
-	if server := os.Getenv("KUBERNETES_MASTER"); len(server) > 0 {
+	if server := os.Getenv("KUBERNETES_CONTROL_PLANE"); len(server) > 0 {
```

Comment:
It's also not clear whether we should add a new envvar that can be used for the same purpose, or point people to specifying a kubeconfig file.

Reply:
It's already deprecated, so I'd suggest not to touch it, we clearly want to drop it. Just make sure we don't mention using that.

Reply:
Yeah, agree that introducing a new variable is not appropriate, and we should simply remove mentions of the use of this and it's deprecation is sufficient (although removing it would break workloads so it's probably not possible in Kube 1.x). 

What was the replacement supposed to be?  Flags?

Reply:
Yes, `--server` flag.